RetailCoder.VBE/UI/ReferenceBrowser/ReferenceBrowser.xaml

```diff
@@ -0,0 +1,76 @@
+﻿<UserControl x:Class="Rubberduck.UI.ReferenceBrowser.ReferenceBrowser"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:Rubberduck.UI.ReferenceBrowser"
+             mc:Ignorable="d" 
+             d:DesignHeight="300" d:DesignWidth="400"
+             d:DataContext="{d:DesignInstance Type=local:ReferenceBrowserViewModel}">
+
+    <TabControl Padding="5">
+
+        <TabItem Header="COM">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition />
+                </Grid.ColumnDefinitions>
+
+                <TextBlock Margin="5"
+                           Padding="5">
+                    Filter:
+                </TextBlock>
+                <TextBox Text="{Binding ComReferencesFilter, UpdateSourceTrigger=PropertyChanged}" 
+                         HorizontalAlignment="Stretch"
+                         Margin="5"
+                         Padding="5"
+                         Grid.Column="1"
+                         Grid.Row="0"/>
+                
+                <local:ReferenceListControl DataContext="{Binding ComReferences}"
+                                            Margin="5"
+                                            Grid.Column="0"
+                                            Grid.Row="1"
+                                            Grid.ColumnSpan="2"/>
+            </Grid>
+        </TabItem>
+
+        <TabItem Header="VBA" >
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition />
+                    <ColumnDefinition Width="Auto"/>
+                </Grid.ColumnDefinitions>
+
+                <TextBox HorizontalAlignment="Stretch"
+                         Margin="5"/>
+
+                <Button Command="{Binding AddVbaProjectReferenceCommand}" 
+                        Grid.Column="1"
+                        Grid.Row="0"
+                        Margin="5"
+                        Padding="5">
+                    Browse...
```

Comment:
Don't hard-code the button text--you have to localize it out to RubberduckUI.resx in the UI folder.  Reference the other .xaml files to see how you reference the value.